docs/resource-providers/amp-provider.md

```diff
@@ -0,0 +1,12 @@
+## Amp Resource Provider
+
+### CreateAmpProvider
+Creates a new AMP Workspace with the provided AMP Workspace name and tags and provides it to the stack.
```

Comment:
You are technically correct. However, customers don't need to be aware of stack and we don't use that noun in the blueprints DSL. 

So let's rename (here and elsewhere) "provides it to the stack" to "makes it available to the blueprint constructs (add-ons, teams, etc.) under the provided name". 

Also keep in mind that a single blueprint can produce multiple stacks (via clone().build()), and that drives the whole reason for resource providers as those are not specific to a single instantiation of a blueprint.